README.md

```diff
@@ -54,6 +54,32 @@ import * as Turbo from '@hotwired/turbo'
 +TurboPower.initialize(Turbo.StreamActions)
 ```
 
+### Installation on a stock Rails 7 install, which utilizes [importmaps](https://github.com/rails/importmap-rails)
+
+On a stock install of Rails 7 modify the following files:
+
+**config/importmaps.rb**
+```diff
+pin "application", preload: true
+- pin "@hotwired/turbo-rails", to: "turbo.min.js", preload: true
++ pin "@hotwired/turbo", to: "turbo.min.js", preload: true
+pin "@hotwired/stimulus", to: "stimulus.min.js", preload: true
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js", preload: true
+```
+
+**app/javascripts/application.js**
+```diff
+- import '@hotwired/turbo-rails'
++ import { Turbo } from '@hotwired/turbo'
```

Comment:
This should work too!
```suggestion
+ import { Turbo } from '@hotwired/turbo-rails'
```

Reply:
I tested that and it doesn't work. Must be `import { Turbo } from '@hotwired/turbo'`.

Reply:
I had this working at some point, but this might even need a fix in turbo itself then. Let's go with your way for now and I will investigate!